docs/connection.md

```diff
@@ -0,0 +1,40 @@
+# Connection
+
+The connection to the server is established via a WebSocket. This page specifies the details of the connection, including the path, authorization, version negotiation, keep-alive, and message encoding.
+
+## Path
+
+The WebSocket connection is established at the following path:
+
+```
+wss://<server>/tachyon
+```
+
+If the server is running on `localhost`, a non-secure `ws://` should be used instead.
+
+## Authorization
+
+The WebSocket connection requires authorization. The full auth process is detailed [here](authorization.md). The `Authorization` header must be set to the access token obtained from the auth process following the `Bearer` scheme.
+
+## Tachyon version negotiation
+
+The client must specify as list of supported tachyon versions in the `Sec-WebSocket-Protocol` header. The server will then select the highest version supported by both the client and server. If the server does not support any of the versions specified by the client, the connection will be closed.
+
+The name of the websocket subprotocol is `{version}.tachyon.beyondallreason.dev`.
```

Comment:
Why does `beyondallreason.dev` need to be part of this string?

Reply:
Quoting WebSockets RFC, emphasis mine:

> These subprotocol names **should** be registered as per Section 11.5. To avoid potential collisions, it is **recommended to use names that contain the ASCII version of the domain name of the subprotocol's originator**. For example, if Example Corporation were to create a Chat subprotocol to be implemented by many servers around the Web, they could name it "chat.example.com".

We don't want to register them realistically, but use the domain name following recommendation.

We don't need to, it's only a recommendation, I can drop it.

Reply:
I think that would be best

Reply:
Removed